backend/src/main/java/site/coduo/common/controller/CommonExceptionHandler.java

```diff
@@ -12,21 +16,27 @@
 
 @Slf4j
 @RestControllerAdvice
-public class CommonErrorController {
+public class CommonExceptionHandler extends ResponseEntityExceptionHandler {
 
-    @ExceptionHandler(NoResourceFoundException.class)
-    public ResponseEntity<ApiErrorResponse> handleNoResourceFoundException(final NoResourceFoundException e) {
-        log.warn(e.getMessage());
+    @Override
+    protected ResponseEntity<Object> handleNoResourceFoundException(final NoResourceFoundException ex,
+                                                                    final HttpHeaders headers,
+                                                                    final HttpStatusCode status,
+                                                                    final WebRequest request
+    ) {
```

Comment:
파라미터 줄바꿈 할 때, 첫번째 파라미터도 줄바꿈 하는 것은 어떨까요??

```suggestion
    protected ResponseEntity<Object> handleNoResourceFoundException(
            final NoResourceFoundException ex,
            final HttpHeaders headers,
            final HttpStatusCode status,
            final WebRequest request
    ) {
```

Reply:
좋습니당!